forager/config.py

```diff
@@ -45,6 +45,11 @@ def _assert_valid_observation_mode(config: ForagerConfig) -> None:
         raise ForagerInvalidConfigException(f'Observation mode must be one of {valid_modes}')
 
 def _maybe_fix_aperture(config: ForagerConfig) -> ForagerConfig:
+    if config.observation_mode == 'world':
```

Comment:
We want to avoid logging a warning if the user has specified a non-ambiguous configuration. For instance, if the user configuration is:
```python
ForagerConfig(
  observation_mode='world',
  aperture=None,
)
```
then there is no reason to warn, everything is configured correctly. We should only warn if we need to change the user's configuration.

Concretely, I would change this line to:
```python
if config.observation_mode == 'world' and config.aperture is not None:
```

Reply:
Okay, will change it now!